project1-sps/script.js

```diff
@@ -0,0 +1,227 @@
+var roll3Number = function () {
+  // generate decimal 0 <= x < 3
+  var randomDecimal = Math.random() * 3;
+
+  // generate integer 0 <= x <= 2
+  var randomInteger = Math.floor(randomDecimal);
+
+  var chosenNumber = randomInteger + 1;
+  return chosenNumber;
+};

```

Comment:
we showed this function logic to you line by line so that its clearer what is going on, but once you understand it, i think its simple enough that it can be condensed to one line

`return Math.floor(Math.random() * 6) + 1;`